examples/service_signin/frontend/service_signin.json

```diff
@@ -0,0 +1,44 @@
+{
+  "base_path": "/log-in-file-self-assessment-tax-return/signin",
+  "content_id": "2b4de3cf-cb13-4e9f-ae37-07090ab1a6e6",
+  "document_type": "service_signin",
+  "first_published_at": "2016-02-29T09:24:10.000+00:00",
+  "locale": "en",
+  "phase": "live",
+  "public_updated_at": "2015-05-28T15:46:51.000+00:00",
+  "publishing_app": "publisher",
+  "rendering_app": "government-frontend",
+  "schema_name": "service_signin",
+  "title": "Self Assessment",
```

Comment:
Since this is a self assessment example, should the filename reflect that?

Reply:
I've seen quite a bit variation in naming the example files; some schemas just have the schema_name.json, others have the content name.  Not sure what the best practice is?